packages/shared/src/features/onboarding/steps/InformativeScreen.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import React from 'react';
+import type { ReactElement } from 'react';
+import classNames from 'classnames';
+import StepHeadline from '../shared/StepHeadline';
+import { Image } from '../../../components/image/Image';
+import { Button, ButtonVariant } from '../../../components/buttons/Button';
+import type { FunnelStepFact } from '../types/funnel';
+
+type InformativeScreenProps = FunnelStepFact & {
+  onTransition: () => void;
+};
+
+const InformativeScreen = ({
+  parameters,
+  onTransition,
+}: InformativeScreenProps): ReactElement => {
+  return (
+    <>
+      {/* Background gradient */}
+      <div
+        className={
+          parameters?.reverse
+            ? 'bg-gradient-purple-reverse'
+            : 'bg-gradient-purple'
+        }
+      />
+      {/* Content container */}
+      <div className="flex min-h-dvh flex-col px-6 pb-20 pt-10">
+        <div
+          data-testid="informative-content"
+          className={classNames(
+            'flex flex-1 items-center gap-6',
+            parameters?.reverse
+              ? 'flex-col-reverse justify-end'
+              : 'flex-col justify-between',
```

Comment:
Along with `parameters.reverse`, we should probably have `parameters.spacing`. Whether it is justify-between, justify-center, or even spaces, etc.

Reply:
For now is out of scope but I think we could add later. Do you think is mandatory now?

Reply:
We can move without it for now but maybe once we introduce another screen we should start considering it.